website/utils/figmaLinks.ts

```diff
@@ -0,0 +1,64 @@
+import { Component } from 'utils/types';
+
+// Not every Component has a corresponding Figma link, but want to be sure we're appropriately keeping track of which components do not
+// @ts-expect-error
+export const figmaLinks: Record<Component, string> = {
```

Comment:
`Partial<Record<Component, string>>` is more accurate

Reply:
Yeah, that's fair.